Translation/Loader/XliffLoader.php

```diff
@@ -59,6 +59,23 @@ public function load($resource, $locale, $domain = 'messages')
                     ->setDesc((string) $trans->source)
                     ->setLocaleString((string) $trans->target)
             ;
+
+            $m->setApproved($trans['approved']=='yes');
+
+            if (isset($trans->target['state'])) {
+                $m->setState($trans->target['state']);
+            }
+
+            if (isset($trans->note)) {
+                if (count($trans->note) > 1) {
+                    foreach ($trans->note as $note) {
+                        $m->addNote((string) $note, isset($note['from']) ? ((string) $note['from']) : null);
+                    }
+                } else {
+                    $m->addNote((string) $trans->note, isset($trans->note['from']) ? ((string) $trans->note['from']) : null);
+                }
```

Comment:
why the test for the count? who cares if its 1 or more? Why not just foreach on $trans->note regardless of 1 or more?